tests/baselines/reference/unionTypeCallSignatures.errors.txt

```diff
@@ -118,15 +114,13 @@ tests/cases/conformance/types/union/unionTypeCallSignatures.ts(73,12): error TS2
     var unionWithOptionalParameter3: { (a: string, b?: number): string; } | { (a: string): number; };
     strOrNum = unionWithOptionalParameter3('hello');
     strOrNum = unionWithOptionalParameter3('hello', 10); // error no call signature

```

Comment:
remove error comment, and edit the next one to say something like `// bad argument type`